src/components/Headings/PageSubtitle.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { Heading } from '@chakra-ui/core';
+
+const PageSubtitle = ({ children, ...rest }) => {
+  return (
+    <Heading as="h2" size="16pt" maxWidth="960px" {...rest}>
```

Comment:
Can we leverage px or rem values for the font size here please :) just for consistency!